library/peerconnection.js

```diff
@@ -174,15 +194,27 @@ class CsioPeerConnection {
     console.log('Channel creating:', label);
     this.datachannels[label] = this.pc.createDataChannel(label);
     this.setChannelCallbacks(label);
+    modCommon.triggerEvent('webrtcEvent',
+      {'pc': this.pc, 'eventLog': JSON.stringify(
+        {'userId': this.userId,'what': 'dataChannelCreated','label': label})
+      });
   }
 
   setChannelCallbacks(label) {
     this.datachannels[label].onmessage = this.handleChannelMessage.bind(this);
     this.datachannels[label].onopen = function(e) {
       console.log('Channel opened:', label);
+      modCommon.triggerEvent('webrtcEvent',
+        {'pc': this.pc, 'eventLog': JSON.stringify(
+          {'userId': this.userId,'what': 'onDataChannelOpened','label': label})
```

Comment:
i think the naming is a bit inconsistent, as some start with "on" and others not

Reply:
I agree. How about we removed prefix "on" from all open/close events ? For example - "dataChannelOpened" instead of "onDataChannelOpened". Any other better alternative naming in your mind ?

Reply:
yes, i think thats fine

Reply:
As mentioned in early comment, this can also be solved by , for example -

``` DataChannel opened for ${this.userId} ```